tests/ops/fixtures/saas/datadog_fixtures.py

```diff
@@ -0,0 +1,89 @@
+from typing import Any, Dict, Generator
+
+import pydash
+import pytest
+from fideslib.core.config import load_toml
+from fideslib.db import session
+from sqlalchemy.orm import Session
+
+from fidesops.models.connectionconfig import (
+    AccessLevel,
+    ConnectionConfig,
+    ConnectionType,
+)
+from fidesops.models.datasetconfig import DatasetConfig
+from fidesops.util.saas_util import load_config
+from tests.ops.fixtures.application_fixtures import load_dataset
+from tests.ops.test_helpers.vault_client import get_secrets
+
+saas_config = load_toml(["saas_config.toml"])
+secrets = get_secrets("datadog")
+
+
+@pytest.fixture(scope="session")
+def datadog_secrets():
```

Comment:
Once you merge in the main branch, you'll see that this takes in a new param of `saas_config`, this let's us import the saas_config once and not have to import it for every SaaS connector fixture.